cryptoapis/src/index.ts

```diff
@@ -3,5 +3,5 @@ import { makeExecute } from './adapter'
 import { makeConfig } from './config'
 
 const NAME = 'CRYPTOAPIS'
-
-export = { NAME, makeExecute, makeConfig, ...expose(util.wrapExecute(makeExecute())) }
+const handlers = expose(util.wrapExecute(makeExecute()))
```

Comment:
Not sure if we are going to have any kind of problems with deployments or node operators with the exposed handlers changed this way. But I couldn't find a way to spread the exposed object to separate exports.